tox.ini

```diff
@@ -319,7 +319,7 @@ commands =
 
 [testenv:lint]
 basepython: python3.8
-recreate = False 
+recreate = True
```

Comment:
What is this for?

Reply:
I have no idea why docs is failing and this was a try at fixing that issue assuming cache was causing the problem. I will check again.

Reply:
@codeboten I remember you did something similar to this in core repo to fix the unusual pipeline failures. I can't get the docs to pass and I couldn't understand error. I would really appreciate some help here.